api_tests/files/views/test_file_detail.py

```diff
@@ -361,3 +361,41 @@ def test_read_only(self):
             expect_errors=True,
             auth=self.user.auth,
         ).status_code, 405)
+
+
+class TestFileTagging(ApiTestCase):
+    def setUp(self):
+        super(TestFileTagging, self).setUp()
+        self.user = AuthUserFactory()
+        self.node = ProjectFactory(creator=self.user)
+        self.file1 = api_utils.create_test_file(
+            self.node, self.user, filename='file1')
+
+    def test_tags_add_and_remove_properly(self):
```

Comment:
I know this was copied almost verbatim from an existing test, but it should _probably_ be broken up into three tests. I'd have felt worse about this if the git blame said my name, but it says Dawn's, so maybe it wasn't entirely my fault.

Also, do file tags generate any logs in general? If so, please check for the logs.